<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>835</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube Download Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="4" colspan="4">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Filter View</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Subscription File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Filter Channel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QComboBox" name="timeFilterBox">
      <item>
       <property name="text">
        <string>24 hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>48 hours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7 days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14 days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31 days</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 year</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="subFileBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QPushButton" name="downloadPathButton">
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5" colspan="2">
     <widget class="QLineEdit" name="downloadPathEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTimeEdit" name="timeStartEdit"/>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_12">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="8">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="2">
        <widget class="QListView" name="downloadedVideoList"/>
       </item>
       <item row="1" column="1">
        <widget class="QListView" name="queuedVideoList"/>
       </item>
       <item row="1" column="0">
        <widget class="QListView" name="availVideoList"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Available Videos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Queued Videos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Downloaded Videos</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QTextBrowser" name="videoBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="subReloadButton">
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Download Window Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Subscription</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="8">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filter time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QComboBox" name="channelFilterBox">
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Scheduling</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QTimeEdit" name="timeEndEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Download Window End</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" colspan="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Start Download</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2" colspan="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Download starting in:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5" colspan="3">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Download Destination</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
